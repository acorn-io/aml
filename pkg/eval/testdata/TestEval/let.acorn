let y: x
x: 0
{
    let x: 1
    b: x
}
c: y
