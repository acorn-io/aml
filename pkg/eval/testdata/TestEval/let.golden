{
  "b": 1,
  "c": 0,
  "x": 0
}